content/docs/other-guides/authentication.md

```diff
@@ -0,0 +1,514 @@
++++
+title = "Kubeflow Authentication with Istio and Dex"
+description = "Instructions for installing Kubeflow with authentication support"
+weight = 4
++++
+
+This installation of Kubeflow is maintained by [Arrikto](https://www.arrikto.com/), is geared towards existing Kubernetes clusters and does not depend on any cloud-specific feature.
+
+In this reference architecture, we use [Dex](https://github.com/dexidp/dex) and [Istio](https://istio.io/) for vendor-neutral authentication.
+
+![platform existing architecture](https://i.imgur.com/OlaN73j.png)
+
+### Prerequisites
+- Kubernetes Cluster with LoadBalancer support.
+
+If you don't have a Kubernetes Cluster, you can create a compliant Kubernetes Engine (GKE) on Google Cloud Platform cluster with the following script:
```

Comment:
Can we delete this? We already have GCP specific instructions that help people create a GKE cluster.

Can we refer people to
The K8s docs if they need help with creating a K8s cluster?

Reply:
done